ledger-core/v2/deploy/kube/README.md

```diff
@@ -0,0 +1,41 @@
+# Kuberenetes deploy manifests
```

Comment:
probably, should be moved to scripts/deploy

Reply:
you mean all deploy dir?

Reply:
yep

Reply:
moved